docs/source/reservations/reservations.rst

```diff
@@ -124,10 +124,10 @@ Example:
 SkyPilot will utilize the reservations similar to AWS reservations as described in :ref:`utilizing-reservations`.
 
 
-GCP Dynamic Workload Scheduler (DWS)
+GCP dynamic workload scheduler (DWS)
```

Comment:
Here, `Dynamic Workload Scheduler` is the name of their product so we should leave it capitalized.